large-message-core/build.gradle.kts

```diff
@@ -31,7 +31,7 @@ dependencies {
     api(group = "org.apache.kafka", name = "kafka-clients", version = kafkaVersion)
 
     implementation(group = "org.slf4j", name = "slf4j-api", version = "2.0.16")
-    val awsVersion = "2.29.4"
+    val awsVersion = "2.29.52"
```

Comment:
Upgrading to 2.30 requires a newer localstack version which breaks version deletion https://github.com/localstack/localstack/issues/12255